telos-frontend/src/components/weather/JournalWeather.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react"
```

Comment:
Good work on implementation of what the team discussed. Can we just follow the warnings please? There's a few of these caused because of the linter.

Reply:
Fixed